apps/src/code-studio/pd/workshop_dashboard/components/survey_results_header.jsx

```diff
@@ -51,16 +52,17 @@ const SurveyResultsHeader = React.createClass({
     this.setState({
       selectedCourse: course,
       filteredWorkshops: filteredWorkshops,
-      selectedWorkshopId: firstWorkshopId
+      selectedWorkshopId: firstWorkshopId,
     });
 
-    this.setSurveyPanel(firstWorkshopId);
+    this.setSurveyPanel(firstWorkshopId, this.getWorkshopFriendlyName(filteredWorkshops[0]));
```

Comment:
For consistency and simplicity it might be worth passing `friendly_name` from the [Workshop model](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/pd/workshop.rb#L163) in the API instead of reconstructing it here.